test/backend/datastores.athena.spec.js

```diff
@@ -0,0 +1,327 @@
+import {assert} from 'chai';
+
+import {connect} from '../../backend/persistent/datastores/Athena';
+
+/* eslint-disable no-invalid-this */
+describe('Athena Unit Tests:', function () {
+    it('connect succeeds', function(done) {
```

Comment:
you can return a `Promise` and `mocha` will do the right thing; like this:

```js
it('connect succeeds', function() {
    return connect(connection).then(conn => {
        assert.isDefined(conn, 'test failed because conn is not defined');
    });
});
```